miner/worker.go

```diff
@@ -1116,11 +1152,18 @@ func (w *worker) commitNewWork(interrupt *int32, noempty bool, timestamp int64)
 			return
 		}
 		header.Coinbase = w.coinbase
+		if useMB {
+			header.Coinbase = maybeMB.Coinbase
```

Comment:
Do we actually override the coinbase or do we just check that coinbase matches what the miner expects? Maybe we allow it to be empty to start as well?

Reply:
override is how I interpreted the OP issue